source/manuals/programming-languages/nodejs.html.md.erb

```diff
@@ -0,0 +1,118 @@
+---
+title: Node.js styleguide
+---
+
+# <%= current_page.data.title %>
+
+For JavaScript at GDS, we use the [standard] styleguide, and that includes for Node.js.
+This gives us consistency across frontend and backend JavaScript implementations, and enables developers to code against the same set of rules wherever their JS is running.
+
+We generally do not use Node.js at GDS for any ‘significant application logic’, however that does not mean we don't use it.
+GOV.UK Pay uses Node to create thin presentation layers that retrieve data from APIs, but do not themselves store data.
+
+However even in these examples, Node applications can include a sizable amount of code.
+So there are additional considerations to be taken into account in order to keep the code readable and maximise maintainability.
+
+## ES2015 and beyond
+One of the best parts of coding Node is that you can use most of the latest JavaScript features.
+This can often (but not always) make for smaller, more easily readable and maintainable code,
+though judgement should be exercised as to when this is the case.
+
+In Node.js applications, it is better to use `const` and `let` rather than `var`.
+
+A useful guide for what ES2015+ features are supported in each version of Node.js can be found at [Node Green].
+
+## Import statements and strict mode
+
+Each `.js` file in your application should be in [strict mode], `'use strict'` should be at the top of each file,
+this will enforce best practices and prevent some potentially unexpected behaviours.
+
+Import statements (i.e. `require('x')`) should always be at the top of each file and should be grouped in the following way:
+1) Built-in modules - Modules that come pre-packaged with Node.js such as `http`, `crypto` and `path`.
+2) NPM dependencies - Modules that you have installed via NPM such as `express`, `request` or `lodash`.
+3) Other application files - Other files the application such as `./config/environment.js`
```

Comment:
This doesn't quite make sense, I think you meant "other files the application **uses**" ?

Reply:
Sorry for not replying sooner. I rewrote this section. You we're right, it was just confusingly written. Hopefully it is clearer now.